DeratControl.Application/Requests/Interfaces/ICommandHandler.cs

```diff
@@ -11,5 +11,24 @@ public interface ICommandHandler<in TRequest> where TRequest : IRequest
     {
         Task<CommandResult> Handle (CommandExecutionContext executionContext, TRequest request);
     }
+    public abstract class BaseCommandHandler<TRequest> : ICommandHandler<TRequest> where TRequest : IRequest
```

Comment:
there is another PR with this class added, please communicate with other developer and push only one class